lib/UA.d.ts

```diff
@@ -18,9 +18,14 @@ export interface CallOptions extends AnswerOptions {
   anonymous?: boolean;
 }
 
+export interface Sockets {
```

Comment:
`Sockets` semantic implies multiple `Socket`s. And this is a different type. You can call it `WeightedSocket`.

Reply:
Also, please move it to WebSocketInterface file as `Socket` definition resides there.

Reply:
And must also be exported in JsSIP.d.ts